.github/pull_request_template.md

```diff
@@ -9,6 +9,7 @@ Following this checklist to help us incorporate your contribution quickly and ea
        is to use the JIRA issue title in the pull request title and in the first line of the commit message.
  - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
  - [ ] Run `mvn clean install apache-rat:check` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
+ - [ ] By default the project packages the tests in an artifact. To activate them,run `mvn clean test -Prun-its`.
```

Comment:
Hi, thanks for your PR.

Two little things

1. you do not activate artifact-bundled tests. At least the `run-its`  does not activate test packaging or similar. Please rephrase to something simpler, e.g. something containing "additional integration tests" and also mention the dev should do this before committing and creating a PR.
2. (nit) space missing after comma.

Thanks!